build/stylesheets/admin-tap.css

```diff
@@ -281,7 +281,9 @@ button.close:active:hover {
 }
 
 /* Hide error field */
-#form-element-wrapper_217 {
+#form-element-wrapper_217,
```

Comment:
When I refactored the TAP form, I deleted field 265 thinking it wasn't needed, and ended up replacing it with field 275. Please update the code to match.

#form-element-wrapper_217,
#form-element-wrapper_271,
#form-element-wrapper_275 {
  visibility: hidden;
  height: 0;
  margin: -20px 0;
}